pkgs/applications/misc/archiver/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildGoPackage
+, fetchFromGitHub
+, lib
+, stdenv
```

Comment:
You don't really need it, it seems? Just keep `lib` here.

Reply:
oh shoot, leftover from go2nix, I’ll remove it.